src/SprykerEco/Yves/Adyen/Dependency/Service/AdyenToUtilEncodingServiceBridge.php

```diff
@@ -29,7 +29,7 @@ public function __construct($encodingService)
      *
      * @return string|null
      */
-    public function encodeJson($value, $options = null, $depth = null): ?string
+    public function encodeJson($value, $options = null, $depth = null)
```

Comment:
return types should be added to all bridges to keep forward compatibility

Reply:
Done.